ompi/mca/topo/base/topo_base_dist_graph_create.c

```diff
@@ -32,9 +32,9 @@ typedef struct _dist_graph_elem {
 } mca_topo_base_dist_graph_elem_t;
```

Comment:
this all looks like whitespace changes in ompi even though this PR is suppose to be for orte.  I'd prefer to see this type of change moved to a separate PR.